src/core/task.c

```diff
@@ -0,0 +1 @@
+#include "task.h"
```

Comment:
Do we need an empty source file? It creates one more translation unit and add to the total linking time.